workers/unity/Assets/Playground/Config/TemplateUtils.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+
+namespace Playground
+{
+    public static class TemplateUtils
```

Comment:
How about renaming this to `EntityTemplateUtils`? The current name is a bit vague.

Reply:
Roger